superset/config.py

```diff
@@ -1289,6 +1289,20 @@ def SQL_QUERY_MUTATOR(  # pylint: disable=invalid-name,unused-argument
     "port": internet_port,
 }
 
+# Configuration for environment tag shown on the navbar. Setting 'text' to '' will hide the tag.
+ENVIRONMENT_TAG_CONFIG = {
+    "variable": "FLASK_ENV",
+    "values": {
+        "development": {
+            "color": "#c73d2e",
+            "text": "Development",
+        },
+        "production": {
+            "color": "#039dfc",
+            "text": "Production",
+        },
```

Comment:
I have a hunch that most people won't want the "Production" label showing up. Their production environment should probably just look... normal, whereas staging/dev/etc would warrant labeling to differentiate them.